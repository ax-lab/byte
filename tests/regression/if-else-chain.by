let sum = 0
for i in 1..9:
	sum = sum + i
	print 'i =', i
	if i == 1:
		print '...is 1'
	else if i == 2:
		print '...is 2'
	else if i == 3:
		print '...is 3'
	else:
		if i % 2 == 0:
			print '...is even'
		else:
			print '...is odd'

print 'sum is', sum
