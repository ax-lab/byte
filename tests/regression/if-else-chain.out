i = 1
...is 1
i = 2
...is 2
i = 3
...is 3
i = 4
...is even
i = 5
...is odd
i = 6
...is even
i = 7
...is odd
i = 8
...is even
i = 9
...is odd
sum is 45
